scrapers/Brazzers.yml

```diff
@@ -1,4 +1,7 @@
+###############################################################################################
```

Comment:
Can you remove these `######` comments?
We don't use them in the rest of the scrapers so its probably better to keep the structure the same for all.